src/Services/Webhooks/Webhooks.API/Controllers/WebhooksController.cs

```diff
@@ -69,10 +69,8 @@ public async Task<IActionResult> SubscribeWebhook(WebhookSubscriptionRequest req
             await _dbContext.SaveChangesAsync();
             return CreatedAtAction("GetByUserAndId", new { id = subscription.Id }, subscription);
         }
-        else
-        {
-            return StatusCode(418, "Grant url can't be validated");
-        }
+
+        return StatusCode(418, "Grant url can't be validated");
```

Comment:
why you have removed this statement from the else block?

Reply:
Hi @erjain. I removed that statement because there is an If block above with a return statement. If it's not obvious, of course, I can return it as it was. The same logic with statements like this applied for example here https://github.com/dotnet-architecture/eShopOnContainers/blob/a339e213ba8aebfaeb88e6c1ad0bb7fc0e80725a/src/Services/Webhooks/Webhooks.API/Controllers/WebhooksController.cs#L80-L93

Reply:
Hi @Ich1goSan, thanks for the clarification, but I believe this condition will be required to return the proper StatusCode when grantOk is false. Let's revert this and proceed with other changes.